cirq-core/cirq/contrib/svg/svg.py

```diff
@@ -25,8 +26,11 @@ def fixup_text(text: str):
 def _get_text_width(t: str) -> float:
     t = fixup_text(t)
     tp = matplotlib.textpath.TextPath((0, 0), t, size=14, prop=FONT)
-    bb = tp.get_extents()
-    return bb.width + 10
+    try:
+        bb = tp.get_extents()
+        return bb.width + 10
+    except:
+        return 21
```

Comment:
where does this magic number come from? consider adding a comment or factoring out into a meaningfully-named global constant

Reply:
The value returned for the CNOT gate in `_get_text_width` is about 21, and this seems like a good enough margin to tell the empty moments apart in the SVG representation.